viadot/sources/outlook.py

```diff
@@ -182,10 +182,13 @@ def _get_messages_from_mailbox(
                     recivers_list = fetched.get("toRecipients")
                     recivers = " "
```

Comment:
Do the `receivers` need to have an empty space or can it be like `receivers=""`?

Reply:
I give an empty space to avoid any problem if finally, the program doesn't have any data to append.

Reply:
If you are sure about this option please add tests for that

Reply:
done!